docker/php-fpm/Dockerfile

```diff
@@ -0,0 +1,122 @@
+FROM php:8.2.10-fpm-alpine
```

Comment:
@mwarzybok-sumoheavy let's use `php:8.2-fpm-alpine` so it always pulls in the latest patch version.